testutil/integration.go

```diff
@@ -0,0 +1,166 @@
+package testutil
+
+import (
+	"strconv"
+
+	sdkerrors "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/evmos/ethermint/crypto/ethsecp256k1"
+	"github.com/evmos/ethermint/encoding"
+
+	"github.com/evmos/evmos/v9/app"
+)
+
+// SubmitProposal delivers a submit proposal tx for a given gov content.
+// Depending on the content type, the eventNum needs to specify submit_proposal
+// event.
+func SubmitProposal(
+	ctx sdk.Context,
+	appEvmos *app.Evmos,
+	pk *ethsecp256k1.PrivKey,
+	content govv1beta1.Content,
+	eventNum int,
+) (id uint64, err error) {
+	accountAddress := sdk.AccAddress(pk.PubKey().Address().Bytes())
+	stakeDenom := stakingtypes.DefaultParams().BondDenom
+
+	deposit := sdk.NewCoins(sdk.NewCoin(stakeDenom, sdk.NewInt(100000000)))
+	msg, err := govv1beta1.NewMsgSubmitProposal(content, deposit, accountAddress)
+	if err != nil {
+		return 0, err
```

Comment:
Nit: you have named return parameters. You could use `return id, err` instead of `return 0, err`